src/main/java/uk/gov/laa/ccms/caab/client/EbsApiClient.java

```diff
@@ -88,6 +88,7 @@ public Mono<CommonLookupDetail> getCommonValues(final String type, final String
             .queryParamIfPresent("code", Optional.ofNullable(code))
             .queryParamIfPresent("description", Optional.ofNullable(description))
             .queryParamIfPresent("sort", Optional.ofNullable(sort))
+            .queryParam("size", 1000)
```

Comment:
This goes towards resolving a couple of bugs in the backlog, CCLS-1740 and CCLS-2016. But I guess we need to cover the other ebs-api calls before we can close them.

Reply:
to be adjusted properly in those stories